scrapy/http/request/__init__.py

```diff
@@ -136,8 +142,43 @@ def from_curl(cls, curl_command, ignore_unknown_options=True, **kwargs):
 
         To translate a cURL command into a Scrapy request,
         you may use `curl2scrapy <https://michael-shub.github.io/curl2scrapy/>`_.
-
-       """
+        """
         request_kwargs = curl_to_request_kwargs(curl_command, ignore_unknown_options)
         request_kwargs.update(kwargs)
         return cls(**request_kwargs)
+
+    def to_dict(self, spider: Optional["scrapy.Spider"] = None) -> dict:
```

Comment:
I wonder if we should make the parameter keyword-only.

Reply:
Sounds good